.github/workflows/release-testflight.yml

```
@@ -0,0 +1,306 @@
+          git config --local user.name "GitHub Action"
+
+      - name: Request CodeRabbit Summary
+        run: |
+          gh pr comment ${{ github.event.pull_request.number }} --body "@coderabbitai review"
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Wait for CodeRabbit Summary
+        id: detect_summary
+        timeout-minutes: 10
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:7:34: Double quote to prevent globbing and word splitting

(shellcheck)